runtime/near-vm-runner/src/wasmer_runner.rs

```diff
@@ -135,7 +137,9 @@ impl IntoVMError for wasmer_runtime::error::RuntimeError {
                 // upon the middleware or backend being used.
                 // As of 0.17.0, thrown only from Singlepass BE and wraps RuntimeError
                 // instance.
-                InvokeError::Breakpoint(_) => VMError::FunctionCallError(WasmTrap(BreakpointTrap)),
+                InvokeError::Breakpoint(_) => VMError::FunctionCallError(
+                    FunctionCallError::Nondeterministic("Wasmer breakpoint".to_string()),
+                ),
```

Comment:
Why break point is nondeterministic error?

Reply:
Because it's not something decided by the contract, I presume.

Reply:
I wonder if we should `unreachable!()` here?